### DISCUSSION TITLE
Unable to intall keda-add-ons-http-controller-manager

### DISCUSSION THREAD

**Discussion by ryzam** [kedacore/charts#155] [Q&A]
Hi guys

I'm unable to install keda-add-ons as one of the components return ErrIamgePull. I installed via Helm Charts

**Comment by tomkerkhove** [comment]
Can you share some more information about the deployment so we can have a look please?

What Helm registry did you pull it from?

**Reply by ryzam** [reply]
Hi @tomkerkhove 

Here is step sequence i did to install and run keda-http

`NAMESPACE="keda"`
`helm`` repo add kedacore https://kedacore.github.io/charts
helm repo update
helm install keda kedacore/keda --namespace ${NAMESPACE} --set watchNamespace=${NAMESPACE}`

Then install http-add-on
`helm install http-add-on kedacore/keda-add-ons-http --create-namespace --namespace ${NAMESPACE}`

![image](https://user-images.githubusercontent.com/597759/119946444-35fc7000-bfc9-11eb-8292-372dc926f5a8.png)

**Reply by tomkerkhove** [reply]
Thanks.

Can you share the YAML of the add-on deployment that it created please?

**Reply by ryzam** [reply]
The add-on deployment createded using helm and i'm not sure any yaml file for that deployment . Where to look it? not familiar with helm installation before this.
